projects/demo/src/app/pages/segmented-control-toggle/segmented-control-toggle-demo.component.ts

```diff
@@ -0,0 +1,73 @@
+import { IdsIconComponent } from '../../../../../widgets/icon/ids-icon.component';
```

Comment:
Should not be relative:
```suggestion
import { IdsIconComponent } from '@i-cell/ids-angular/icon';
```